crates/re_types_builder/src/objects.rs

```diff
@@ -1196,6 +1189,10 @@ impl Attributes {
                 .unwrap(),
         )
     }
+
+    pub fn has(&self, name: impl AsRef<str>) -> bool {
+        self.0.contains_key(name.as_ref())
```

Comment:
We should also check the here, or we will get confused users writing `nullable=false` and wondering why the output is still nullable. Then `is_set` is probably a better name.